hil-test/tests/esp_wifi_floats.rs

```diff
@@ -0,0 +1,193 @@
+//! Cp0Disable exception regression test
+
+//% CHIPS: esp32 esp32s2 esp32s3
+//% FEATURES: esp-wifi esp-alloc
+//% FEATURES: esp-wifi esp-alloc xtensa-lx-rt/float-save-restore
```

Comment:
Let's hope CI picks up the first feature set, otherwise this needs to be removed.